src/Pim/Bundle/ApiBundle/tests/integration/Controller/Product/SuccessLargeAndOrderedListProductIntegration.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Pim\Bundle\ApiBundle\tests\integration\Controller\Product;
+
+use Doctrine\Common\Collections\Collection;
+use Pim\Component\Catalog\Model\ProductInterface;
+use Pim\Component\Catalog\tests\integration\Normalizer\NormalizedProductCleaner;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * We want to test the API is capable of returning an ordered list of 100 items.
+ * ie, twice the size of a cursor page
+ */
+class SuccessLargeAndOrderedListProductIntegration extends AbstractProductTestCase
+{
+    /** @var Collection */
+    private $products;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if (1 === self::$count) {
```

Comment:
However, there is only 1 test, so it can avoid refactoring when merging master into.